junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/execution/InvocationInterceptorChain.java

```diff
@@ -137,6 +147,11 @@ private void fail(String prefix) {
 			throw new JUnitException(prefix + ": " + commaSeparatedInterceptorClasses);
 		}
 
+		@Override
+		public void skip() {
+			LOG.debug(() -> "The invocation is skipped");
+			invoked.set(true);
```

Comment:
:x: We should rename this to `invokedOrSkipped`.

Reply:
And it should call `delegate.skip()`.

Reply:
done